cookbook/integrations/external_services/databricks/sandbox.config

```diff
@@ -0,0 +1,2 @@
+[sdk]
+workflow_packages=databricks
```

Comment:
can we get rid of this file?  Tests will break right?

Reply:
We can't for now, Tests will break